### PR TITLE
proposing Tim as PMC member

### PR DESCRIPTION
**Pull request by tuxmea**

### CODE REVIEW

**Comment by tuxmea** [comment]
I can not add @bastelfreak as reviewer.

File: share/elections/2023-01.md [hunk 1]
```diff
@@ -65,12 +65,13 @@ E.g:
...
 
-| Candidate                    | Presentation       | Link          |
-| ---------------------------- | ------------------ | ------------- |
-|  |  |  |
+| Candidate                    | Presentation        | Link          |
+| ---------------------------- | ------------------- | ------------- |
+| [Tim Meusel][tm]             | [Presentation][tmp] |               |
 
+[tm]:https://github.com/bastelfreak
+[tmp]:2023-01/bastelfreak.md
```
Comment (ekohl) [inline share/elections/2023-01.md:73]: I think you could even use 
```suggestion
| [Tim Meusel]             | [Presentation][tmp] |               |

[Tim Meusel]:https://github.com/bastelfreak
[tmp]:2023-01/bastelfreak.md
```

File: share/elections/2023-01/bastelfreak.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Tim Meusel
+
+*« Deeply involved into voxpupuli »*
+
+Hi,
+
+I would like to propose Tim Meusel as PMC member.
```
Comment (ekohl) [inline share/elections/2023-01/bastelfreak.md:7]: I is a bit ambiguous in this document. Perhaps include your own name?

**Review by ekohl — APPROVED** [review]

**Review by bastelfreak — APPROVED** [review]
I accept the nomination!

**Pull request merged**